public/js/index.js

```diff
@@ -0,0 +1,62 @@
+(function() {
+
+  // The client ID/secret is set up here:
+  // https://oauth-stable.dev.lcip.org/console/client/487e639f210d112d
+  // It requires the site to be running from http://pay.dev
+
+  // WARNING: this is just an example app. In a real app you wouldn't put your client
+  // secret in the client code.
+  var clientSecret = "6105ec8d4294a4d0cd8e8b8e0bc016eb8ba6eaf1aa0baff059af828a702bfe4b";
+
+  var fxaRelierClient = new FxaRelierClient("487e639f210d112d", {
+    contentHost: 'https://stable.dev.lcip.org',
+    oauthHost: 'https://oauth-stable.dev.lcip.org/v1',
+    clientSecret: clientSecret,
+  });
+
+  $('button#signin').on('click', function() {
+    console.log('signing in');
+
+    fxaRelierClient.auth.signIn({
+      state: null,  // ?
+      ui: 'lightbox',
+      redirectUri: 'http://pay.dev/',
+      scope: 'profile payments'
+    })
+    .then(function (result) {
+      console.log('login succeeded:', result);
+
+      console.log('requesting token for code', result.code);
+
+      // This produces a CORS error for some reason:
```

Comment:
my bad, I realized that this function was meant to be run server side only since it requires the client secret. I thought it was overkill to run a server just to hide the secret (since it doesn't really matter for the example app). I removed all this code.